.wordlist.txt

```diff
@@ -0,0 +1,291 @@
+accessmethod
+accessmethoddescriptor
+accessmethods
+actiondescriptor
+additionalresource
+addversion
+adr
+aggregative
+aml
+anchore
+api
+archlinux
+artifactid
+aur
+auth
+autoconfigure
+aws
+backend
+backends
+basename
+bindata
+blobaccess
+blobhandler
+bool
+boolean
+buildx
+cas
+cli
+cliconfig
+clis
+closable
+closeable
+cmds
+cncf
+codeql
+commontransportarchive
+componentaccess
+componentaccessimpl
+componentarchive
+componentreferences
+componentversionaccess
+componentversionaccessimpl
+componentversions
+compvers
+config
+configfile
+configs
+consumerid
+containerd
+crd
+creationtime
+credentialrequest
+credentialsspec
+crossplatform
+crypto
+ctf
+ctfs
+ctx
+cvs
+deepcopy
+defs
+dependabot
+deployscript
+deserializable
+deserialization
+deserialize
+deserialized
+dest
+digesters
+directorytree
+dirtree
+dns
+dockerconfig
+downloadconstraint
+downloader
+downloaderdescriptor
+downloaders
+downloadhandlers
+dry-run
+dup
+dups
+dynaml
+echoserver
+entrypoint
+executables
+executoroptions
+executorspecification
+extensibility
+filepath
+filesystem
+filesytem
+finalizer
+finalizers
+foldername
+forwardspec
+gapped
+gav
+getinexpensivecontentversionidentity
+getresourcesbyname
+getter
+ghcr
+github
+gitignore
+gitops
+globalaccess
+globals
+gocritic
+gofmt
+golang
+golangci
+gomega
+goreleaser
+gosec
+goutils
+groupid
+gzip
+handleoutput
+hashers
+hashicorp
+hashicorpvault
+helmchart
+helmchartrepository
+helmdemo
+helmexecutor
+helminstaller
+helmrepository
+hoc
+homebrew
+hostname
+hostpath
+html
+http
+https
+iaas
+imagemapping
+inlined
+installable
+instantiation
+io
+ioutils
+iterable
+json
+jsonschema
+keyless
+keypair
+keyring
+kubebuilder
+kubeconfig
+kubernetes
+labelmergespecification
+lifecycle
+linter
+linters
+lister
+listers
+localizations
+logconfig
+logrus
+lookups
+macOS
+makefile
+makefiles
+mandelsoft
```

Comment:
Don't we try to make the repository independent of user-specifc implementations?

Instead of adding the word to the wordlist, we could use the found occurrences to achieve our goal.

Reply:
yes and no... afaik (my knowledge is 2 months old), Uwe moved a few more things under his account and Gerald agreed and said it's ok. So it looks to me, we won't be able to get rid of user-specifc implementations.

Reply:
Okay, thats how it is.